infrastructure/assumptions/resources/exists.txt

```diff
@@ -0,0 +1 @@
+I EXIST!
```

Comment:
I might have put this in `/common`, or even just in `/infrastructure`

Reply:
I can move it.